[question] Are there any plans to enhance command `pouch log`?

### Ⅰ. Issue Description

Now, `pouch log` command is unavailable.

```
pouch logs c534fd
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xd7b820]

goroutine 1 [running]:
main.(*LogsCommand).runLogs(0xc42007fc80, 0xc42052e050, 0x1, 0x1, 0x0, 0x0)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/cli/logs.go:71 +0xf0
main.(*LogsCommand).Init.func1(0xc4201e0d80, 0xc42052e050, 0x1, 0x1, 0x0, 0x0)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/cli/logs.go:40 +0x48
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).execute(0xc4201e0d80, 0xc42052e020, 0x1, 0x1, 0xc4201e0d80, 0xc42052e020)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:742 +0x48e
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc4201ef200, 0xd72881, 0xc4201ef200, 0xc42051def0)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:827 +0x2e4
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4201ef200, 0xc4201e1680, 0xc4201e1680)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:780 +0x2b
main.(*Cli).Run(0xc4201555e0, 0xfb5de0, 0xc42051f620)
	/tmp/pouchbuild/src/github.com/alibaba/pouch/cli/cli.go:78 +0x2f
main.main()
	/tmp/pouchbuild/src/github.com/alibaba/pouch/cli/main.go:49 +0x952
```

### Ⅱ. Describe what happened


### Ⅲ. Describe what you expected to happen


### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅴ. Anything else we need to know?


### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/cli/logs.go
+++ b/cli/logs.go
@@ -63,15 +63,6 @@ func (lc *LogsCommand) runLogs(args []string) error {
 	ctx := context.Background()
 	apiClient := lc.cli.Client()
 
-	c, err := apiClient.ContainerGet(ctx, containerName)
-	if err != nil {
-		return err
-	}
-
-	if !validDrivers[c.HostConfig.LogConfig.Type] {
-		return fmt.Errorf("\"logs\" command is supported only for \"json-file\" and \"journald\" logging drivers (got; %s)", c.HostConfig.LogConfig.Type)
-	}
-
 	opts := types.ContainerLogsOptions{
 		ShowStdout: true,
 		ShowStderr: true,

